### ISSUE TITLE
This version of Windows does not support ANSI colors

### ISSUE THREAD

**Issue by nu8** [bootandy/dust#87]
If I run `dust`, I get the expected output, but also this:

~~~
This version of Windows does not support ANSI colors, setting no_color flag
~~~

this even happens with `-c, --no-colors`, I would think that one of those
invocations should not have the warning.

**Comment by bootandy** [comment]
Yes, I think I have a fix for that on a branch somewhere, but you are correct. I'll fix it on the next release.

**Comment by bootandy** [comment]
shipped in 0.5.2
